### ISSUE TITLE
Generating lots of empty (and unremovable) comments

### ISSUE THREAD

**Issue by iamjohnbarker** [Aurelain/react-style-editor#2]
Really loving this project so far and I think it will slot in nicely for a project of mine.

While testing, I have noticed that I have made lots of empty comments and I just cannot seem to get rid of them in the editor.

<img width="426" alt="Screenshot 2021-12-26 at 19 37 40" src="https://user-images.githubusercontent.com/7009061/147417188-9fdf9c10-a834-4965-bab4-1a5ac408a806.png">

Is there a keyboard shortcut I am missing, or is there a rule you (or I) could add to the code that will auto-delete an empty comment?

**Comment by Aurelain** [comment]
Thanks for the report, it does seem to be a bug:
```css
foo{bar;/*x*/}
```
However, the following input doesn't manifest the bug:
```css
foo{bar;/*x;*/}
```
I will investigate the issue in the following days.

**Comment by iamjohnbarker** [comment]
Wonderful - thanks for taking a look

**Comment by Aurelain** [comment]
Resolved in [v0.4.0](https://github.com/Aurelain/react-style-editor/releases/tag/v0.4.0) by allowing to toggle any comment.

Fun fact: Even the Chrome DevTools has issues with comments. If you try to toggle the following declarations in Chrome, messy issues arise:
```html
<div style='background:/*x*/lime;color:red/*x*/;'>hello</div>
```
![image](https://user-images.githubusercontent.com/2912123/147563847-0d271f78-863d-4e1c-bc5e-0116849bc7ef.png)
